library/cartridge_get_control_instance.py

```diff
@@ -84,17 +85,16 @@ def get_twophase_commit_versions(control_console, advertise_uris):
     return versions, err
 
 
-def candidate_is_ok(uri, names_by_uris, module_hostvars):
+def candidate_is_ok(uri, names_by_uris, module_hostvars, cluster_disabled_instances):
     instance_name = names_by_uris[uri]
-    instance_vars = module_hostvars[instance_name]
-
-    if helpers.is_expelled(instance_vars):
+    if instance_name in cluster_disabled_instances:
```

Comment:
`cluster_disabled_instances` name confuses when used before `helpers.is_enabled`. Names don't reflect that some instances are currently disabled, and others should be disabled.

Reply:
helpers.is_enabled works only with inventory, we can make helpers.is_enabled_in_inventory